models/review/src/generator.ts

```diff
@@ -0,0 +1,38 @@
+import { ClientLogging } from '@commercetools-test-data/commons';
+import { sequence, fake, Generator } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import { TReview } from './types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+// https://docs.commercetools.com/api/projects/reviews#reviewdraft
```

Comment:
Should this be:
```suggestion
// https://docs.commercetools.com/api/projects/reviews#review
```
?